### DISCUSSION TITLE
How to access other legacy vuex getters from pinia store

### DISCUSSION THREAD

**Discussion by natemate90** [vuejs/pinia#1306] [Help and Questions] [answered]
Hi,

I was wondering if this is the correct way to access vuex getters while migrating from Vuex to Pinia:
<img width="1008" alt="Screenshot 2022-05-18 at 17 43 27" src="https://user-images.githubusercontent.com/18378547/169085953-7dd7cbbf-ab5d-48cf-b37d-cfe26c45909f.png">
This seems to work, but I couldn't find something in the official docs.
Does anyone happen know how to resolve the typescript error: `Property '$nuxt' does not exist on type...` ?

**Comment by posva** [comment, marked as answer]
If you are using the nuxt plugin, access `$nuxt` through `this`:

```ts
getters: {
  productType(): <type annotation needed here> {
    return this.$nuxt //...
  }
}
```

**Reply by natemate90** [reply]
Thanks!
